onnxruntime/python/tools/transformers/README.md

```diff
@@ -9,12 +9,12 @@ This tool can help in the following senarios:
 * Disable or enable some fusions to see its impact on performance or accuracy.
 
 ## Installation
+
 First you need install onnxruntime or onnxruntime-gpu package for CPU or GPU inference. To use onnxruntime-gpu, it is required to install CUDA and cuDNN and add their bin directories to PATH environment variable.
 
-This tool can be installed using pip:
-```console
-pip install --upgrade onnxruntime-tools
-```
+## Limitations
+
+Due to CUDA implementation of Attention kernel, maximum hidden dimension is 4096 for float16 model and 2048 for float32 model in GPU. Normally, maximum supported sequence length is 4096 for Longformer and 1024 for other types of models.
```

Comment:
It's easy to miss documentation. Is it possible to return/print an appropriate error when the code runs? The error can point to this documentation if it's easier.

Reply:
Yes. I'll do it in another PR.